### ISSUE TITLE
make `trigger_client_event` return the response instead of None

### ISSUE THREAD

**Issue by lolrenx** [adamchainz/django-htmx#219]
### Description

Currently, `trigger_client_event` returns None after adding the custom event name and params to the response header. 

```
def view(response):
...
response = render(request, self.template_name , context=context)
trigger_client_event(
    response=response,
    name="showMessage",
    params={
        "value": message
    }
)
return response
```

making it return the modified response would imo allow for a more elegant view code (or even maybe add multiple events) : 

```
def view(response):
...
response = render(request, self.template_name , context=context)
return trigger_client_event(
    response=response,
    name="showMessage",
    params={
        "value": message
    }
)
```

**Comment by adamchainz** [comment]
I agree!

The API is copied from Django's functions like [`add_never_cache_headers`](https://docs.djangoproject.com/en/4.0/ref/utils/#django.utils.cache.add_never_cache_headers). This seems like a good small change to suggest upstream as well.

**Comment by lolrenx** [comment]
Thank you! I'd gave sent you a PR but you beat my to it. Looking forward to decommissioning my kludgy wrapper!

**Comment by adamchainz** [comment]
released in 1.10.0
